cmd/mesh/manifest-apply.go

```diff
@@ -35,8 +35,9 @@ type manifestApplyArgs struct {
 	readinessTimeout time.Duration
 	// wait is flag that indicates whether to wait resources ready before exiting.
 	wait bool
-	// yes means don't ask for confirmation (asking for confirmation not implemented)
-	yes bool
+	// skipConfirmation means whether skip for confirmation when manifest apply or not
+	// if skipConfirmation=true, it means skip for confirmation; otherwise, ask for confirmation
```

Comment:
skipConfirmation determines whether the user is prompted for confirmation. If set to true, the user is not prompted and a Yes response is assumed in all cases.